modules/imgproc/src/pyramids.cpp

```diff
@@ -344,15 +344,15 @@ template<> int PyrDownVecH<ushort, int, 4>(const ushort* src, int* row, int widt
 template<> int PyrDownVecH<float, float, 1>(const float* src, float* row, int width)
 {
     int x = 0;
-    const float *src0 = src, *src2 = src + 2, *src4 = src + 4;
+    const float *src0 = src, *src2 = src + 2, *src4 = src + 3;
```

Comment:
ASAN problem is fixed. Thanks!

I believe we should adjust "src" names here (they are confusing).